internal/bundlegen/generateapi.go

```diff
@@ -617,10 +617,8 @@ func getQuotaDefinition(i interface{}) (quotaDef, error) {
 	quota := quotaDef{}
 	jsonMap := map[string]string{}
 	str := fmt.Sprintf("%s", i)
-
-	if err := json.Unmarshal([]byte(str), &jsonArrayMap); err != nil {
-		return quotaDef{}, err
-	}
+	fmt.Println(str)
```

Comment:
Remove "fmt.Println(str)" or change to "clilog.Debug.Println("Quota map: ", str)"